docs/source/index.rst

```diff
@@ -83,6 +83,8 @@ Sampling
 
    - :class:`RejectionABC`, requires a :class:`LogPrior` that can be sampled
      from and an error measure.
+   - :class:`ABCSMC`, requires a :class:`LogPrior` that can be sampled from
+     from and an error measure.
```

Comment:
Missed this the last time, but for both ABCs this should be:
- an :class`ErrorMeasure`
- an :class`ErrorMeasure` to ...